constraint/pkg/client/drivers/k8scel/schema/schema.go

```diff
@@ -16,7 +17,8 @@ import (
 const (
 	// Name is the name of the driver.
 	Name           = "K8sNativeValidation"
-	ReservedPrefix = "g8r-"
+	ReservedPrefix = "g8r_"
+	ParamsName     = "params"
```

Comment:
would be good to add few comments for these new const so we know how they are intended to be used